docs/builds/guides/frameworks/react.md

```diff
@@ -67,18 +67,73 @@ The `<CKEditor>` component supports the following properties:
 * `editor` (required) &ndash; The {@link module:core/editor/editor~Editor `Editor`} constructor to use.
 * `data` &ndash; The initial data for the created editor. See the {@link builds/guides/integration/basic-api#interacting-with-the-editor Basic API} guide.
 * `config` &ndash; The editor configuration. See the {@link builds/guides/integration/configuration Configuration} guide.
-* `onInit` &ndash; A function called when the editor was initialized. It receives the initialized {@link module:core/editor/editor~Editor `editor`} as a parameter.
+* `id` &ndash; The editor ID. When that property changes the component restarts the editor with new data instead of setting it on an initialized editor.
+* `onReady` &ndash; A function called when the editor is ready with an {@link module:core/editor/editor~Editor `editor`} instance. This callback can be called after the initialization and after restarts of the editor.
 * `disabled` &ndash; A Boolean value. The {@link module:core/editor/editor~Editor `editor`} is being switched to read-only mode if the property is set to `true`.
 * `onChange` &ndash; A function called when the editor data has changed. See the {@link module:engine/model/document~Document#event:change:data `editor.model.document#change:data`} event.
 * `onBlur` &ndash; A function called when the editor was blurred. See the {@link module:engine/view/document~Document#event:blur `editor.editing.view.document#blur`} event.
 * `onFocus` &ndash; A function called when the editor was focused. See the {@link module:engine/view/document~Document#event:focus `editor.editing.view.document#focus`} event.
-* `onError` &ndash; A function called when the editor has crashed during the initialization. It receives the error object as a parameter.
+* `onError` &ndash; A function called when the editor has crashed during the initialization or during the runtime. It receives the error instance and the error details (`phase` and `willEditorRestart`) as arguments.
 
-The editor events callbacks (`onChange`, `onBlur`, `onFocus`) receive two parameters:
+The editor events callbacks (`onChange`, `onBlur`, `onFocus`) receive two arguments:
 
 1. An {@link module:utils/eventinfo~EventInfo `EventInfo`} object.
 2. An {@link module:core/editor/editor~Editor `Editor`} instance.
 
+### Context feature
+
+The `ckeditor5-react` integration comes with a ready-to-use component for the {@link features/context-and-collaboration-features Context feature} that is useful when used together with some {@link features/collaboration CKEditor 5 collaboration features}.
+
+```jsx
+import React, { Component } from 'react';
+import { CKEditor, CKEditorContext } from '@ckeditor/ckeditor5-react';
+
+// Assuming that application builds the editor from source.
+import Context from '@ckeditor/ckeditor5-core/src/context';
+import ClassicEditor from './ckeditor';
+
+class App extends Component {
+	render() {
+		return (
+			<div className="App">
+				<CKEditorContext context={ Context }>
```

Comment:
I tried to pass an object as the config:

```js
const editorConfiguration = {
	plugins: [ Essentials, Bold, Italic, Paragraph ],
	toolbar: [ 'bold', 'italic' ]
};
```

And I ended with an error:

```
index.js:1 CKEditorError: context-initplugins-invalid-plugin: Only a plugin marked as a context plugin is allowed to be used with a context. Read more: https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/error-codes.html#error-context-initplugins-invalid-plugin
 {}
    at Context.initPlugins (http://localhost:3000/static/js/0.chunk.js:3959:15)
    at http://localhost:3000/static/js/0.chunk.js:4109:23
    at new Promise (<anonymous>)
    at Function.create (http://localhost:3000/static/js/0.chunk.js:4107:12)
    at Or._creator (http://localhost:3000/static/js/main.chunk.js:1218:181)
    at http://localhost:3000/static/js/main.chunk.js:1287:79 {phase: "initialization", willContextRestart: false}
```

We should explain which plugins are accepted by the context feature and why we can't provide a list of plugins that should be used for all editors. Also, a way how to do that would be nice.

Reply:
This is up to the Context guide. I wouldn't duplicate the information here. This section is rather designed for people who know the CKEditor 5 Context feature.

Reply:
So we could add a link to the context feature that described the options.

Reply:
I checked these docs and they lack some important parts...

Reply:
Also, the problem here is that currently, only collaboration plugins are the context plugins AFAIK so it's hard to create a snippet with private plugins.